### PR TITLE
openEnds in Maridia

### PR DESCRIPTION
**Pull request by DIProgan**

### CODE REVIEW

File: region/maridia/inner-pink.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
           "sparking": {
             "runways": [
               {
-                "length": 2,
+                "length": 13,
```
Comment (fdtaillefer) [inline region/maridia/inner-pink.json]: This one would need a reset, right?

Comment (DIProgan) [reply region/maridia/inner-pink.json]: Nah it's a respawn block

File: region/maridia/inner-yellow.json [hunk 2]
```diff
@@ -18,14 +18,32 @@
                 "length": 5,
                 "requires": null
               }
+            ],
+            "canLeaveCharged": [
```
Comment (fdtaillefer) [inline region/maridia/inner-yellow.json]: Gonna have to explain that one, I can't figure it out

File: region/maridia/inner-pink.json (hunk 1 above)
Comment (fdtaillefer) [reply region/maridia/inner-pink.json]: Golly, you're right

**Pull request merged**